docs/docs/user/getting-started/components.md

```diff
@@ -0,0 +1,529 @@
+      # have any prefix you can specify an empty string.
+      prefix: ${pipeline_name}-
+      # Helm repository configuration
+      repoConfig:
+        repositoryName: my-repo # required
+        url: https://bakdata.github.io/ # required
+        repoAuthFlags: 
+          username: user
+          password: pass
+          ca_file: /home/user/path/to/ca-file
+          insecure_skip_tls_verify: false
+      version: 1.0.0
+    ```
+
+### deploy
+### destory
+### reset
+### clean
+
+## KafkaApp
+
+_KafkaApp_ inherits from [_KubernetesApp_](#kubernetesapp).
+
+### Usage
+
+Inherits from KubernetesApp. Often used in `defaults.yaml` Not usually used to deploy Kafka applications as they almost could be defined as either a _StreamsApp_ or a _Producer_.
+
+
+### Configuration
+
+??? "`pipeline.yaml`"
+
+    ```yaml
+      # Base component for Kafka-based components.
+      # Producer or streaming apps should inherit from this class.
+      - type: kafka-app # required
+        name: kafka-app # required
+        namespace: namespace # required
+        # `app` can contain application-specific settings, hence  the user is free to
+        # add the key-value pairs they need. The fields that are specific to kafka
+        # apps are marked with `# kafka-app-specific`
+        app: # required
+          streams: # required, kafka-app-specific
+            brokers: ${broker} # required
+            schemaRegistryUrl: ${schema_registry_url}
+          nameOverride: override-with-this-name # kafka-app-specific
+          key1:
+            key4: value3
+          key2: value1
+          key3: value2
+        # Topic(s) from which the component will read input
+        from:
+          topics: # required
+            ${pipeline_name}-input-topic:
+              type: input # required
+              role: topic-role
+        # Topic(s) into which the component will write output
+        to:
+          topics: # required
+            ${pipeline_name}-output-topic:
+              type: output # required
+              role: topic-role
+              keySchema: key-schema
+              valueSchema: value-schema
+              partitions_count: 1
+              replication_factor: 1
+              configs: 
+                config1: config1
+          models: 
+            model: model
+        # Pipeline prefix that will prefix every component name. If you wish to not 
+        # have any prefix you can specify an empty string.
+        prefix: ${pipeline_name}-
+        # Helm repository configuration, the default value is given here
+        repoConfig:
+          repositoryName: bakdata-streams-bootstrap # required
+          url: https://bakdata.github.io/streams-bootstrap/ # required
+          repo_auth_flags:
+            username: user
+            password: pass
+            ca_file: /home/user/path/to/ca-file
+            insecure_skip_tls_verify: false
+        version: "2.7.0"
+    ```
+
+
+### deploy
+### destory
```

Comment:
```suggestion
### destroy
```

As there is no different behavior for deploy/destroy than in the super class, I would remove it

Reply:
removed

Reply:
I agree, referring to where it's implemented is a good suggestion